tests/test_command_handler.py

```diff
@@ -207,7 +241,7 @@ async def test_stop(self):
         await asyncio.sleep(0.5)
         self.assertFalse(self.command_handler._started)
 
-    async def test_get_telemetrey(self):
+    async def test_get_telemetry(self):
```

Comment:
I am a bit confused how test_start, test_stop, and test_get_telemetry work, when naively it looks to me as if you are making real devices with real sensors, not mock devices and mock sensors.

If the answer is at all subtle then perhaps put a doc string for the whole test class, or for one of those test_ methods?

Reply:
This is indeed one of the things that I don't like about my code. I start in simulation mode (which ensures that a MockDevice is used) but still a full fledged configuration is needed because of the configuration validation. The main intention here is to test multiple MockDevices at the same time. I've added doc strings and inline comments.

Reply:
I also simplified the `test_configure` method since it had duplicate tests.